demos/button.html

```diff
@@ -74,9 +74,7 @@
     <header class="mdc-toolbar mdc-toolbar--fixed">
       <div class="mdc-toolbar__row">
         <section class="mdc-toolbar__section mdc-toolbar__section--align-start">
-          <span class="catalog-back">
```

Comment:
We shouldn't be removing this span to change the `padding-right`.

Reply:
Why not removing the `span`? I kept the class :smiley:

Reply:
I guess it's not needed anymore :)